source/clear-linux/tutorials/machine-learning/machine-learning.rst

```diff
@@ -23,12 +27,12 @@ Before you install any new packages, update |CL| with the following command:
 
    sudo swupd update
 
-Once your system is updated, add the following bundles to your system:
+After your system is updated, add the following bundles to your system:
 
-* `machine-learning-web-ui`: This bundle contains the Jupyter application.
+* :file:`machine-learning-web-ui`: This bundle contains the Jupyter application.
```

Comment:
Hey @krisdale, can you confirm that you're standardizing use of `:command:` for bundle names? 
I noticed that you used that directive in [developer-workstation](https://github.com/clearlinux/clear-linux-documentation/blob/master/source/clear-linux/guides/maintenance/developer-workstation.rst) and several other guides.  I know we've had to manage a lot of moving parts lately.

Reply:
@DougTW. Please change the role from `:file:` to `:command:` for all bundles. 
Also, this is very minor. Please capitalize references to Figures.